### PR TITLE
bug: fix documentation for CoreAvatar 

### PR DESCRIPTION
**Pull request by PritamIT2023**
## Description

Clearly describe the changes you've made in this pull request. Explain the purpose and reasoning behind the changes. Reference any relevant issues or discussions using keywords like "Fixes #<issue_number>" or "Resolves #<issue_number>".

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: app/components/component-docs/dataDisplay/CoreAvatar.docs.js [hunk 1]
```diff
@@ -85,11 +90,9 @@ export default function CoreAvatarDocs() {
           >
             <CoreAvatar>W</CoreAvatar>
```
Comment (techoneel) [inline app/components/component-docs/dataDisplay/CoreAvatar.docs.js:91]: Why do we change two letter words to single?

File: app/components/component-docs/dataDisplay/CoreAvatar.docs.js [hunk 2]
```diff
@@ -248,18 +250,20 @@ export default function CoreAvatarDocs() {
...
         }
-        code={`
-<CoreStack styleClasses={[CoreClasses.ALIGNMENT.JUSTIFY_CONTENT_CENTER]} direction="row" spacing={2}>
+        code={`<CoreStack styleClasses={[CoreClasses.ALIGNMENT.JUSTIFY_CONTENT_CENTER]} direction="row" spacing={2}>
   <CoreAvatar
     styleClasses={[CoreClasses.BG.BG_PRIMARY]}
     alt="Remy Sharp"
     src="/broken-image.jpg"
-  >B</CoreAvatar>
+  >W</CoreAvatar>
```
Comment (techoneel) [inline app/components/component-docs/dataDisplay/CoreAvatar.docs.js:266]: The requirement is to show the first letter of the alt text will show if the src is broken.

**Review by techoneel — CHANGES_REQUESTED** [review]

**Comment by techoneel** [comment]
Please see why the branch name got impacted.

File: app/components/component-docs/dataDisplay/CoreAvatar.docs.js (hunk 1 above)
Comment (PritamIT2023) [reply app/components/component-docs/dataDisplay/CoreAvatar.docs.js:91]: Changed letter due to keep similar to MUI Avatar documentation.

**Comment by PritamIT2023** [comment]
> 
> 
> Please see why the branch name got impacted.

I don't see any branch name validation error in PR-Checks

**Comment by wrappidcare** [comment]
Merging since stale over 24hrs.

**Comment by wrappidcare** [comment]
Merging since stale over 24hrs.

**Comment by wrappidcare** [comment]
Merging since stale over 24hrs.

**Comment by wrappidcare** [comment]
Merging since stale over 24hrs.

**Comment by wrappidcare** [comment]
Merging since stale over 24hrs.

**Review by techoneel — APPROVED** [review]
good to go

**Pull request merged**